openpype/hosts/houdini/api/pipeline.py

```diff
@@ -450,3 +466,33 @@ def main_take(no_update=True):
             instance_node.bypass(not new_value)
     except hou.PermissionError as exc:
         log.warning("%s - %s", instance_node.path(), exc)
+
+
+class HoudiniProjectSettings:
+    def __init__(self, project_name=""):
```

Comment:
```suggestion
    def __init__(self, project_name=get_current_project_name()):
```
Is it going to be used in somewhere else? If not, implement` from openpype.pipeline import get_current_project_name` and use `get_current_project_name()` . Otherwises the system would give the KeyError below
```
Installing OpenPype ...
Setting scene FPS to 25
>>> [ *** Discovered 1 abstract plugins
- AutoCreator ] 
  - { CreateArnoldRop }: [  No settings found for CreateArnoldRop  ] 
  - { CreateBGEO }: [  No settings found for CreateBGEO  ] 
  - { CreateHDA }: [  No settings found for CreateHDA  ] 
  - { CreateKarmaROP }: [  No settings found for CreateKarmaROP  ] 
  - { CreateMantraROP }: [  No settings found for CreateMantraROP  ] 
  - { CreateRedshiftProxy }: [  No settings found for CreateRedshiftProxy  ] 
  - { CreateReview }: [  No settings found for CreateReview  ] 
  - { CreateVrayROP }: [  No settings found for CreateVrayROP  ] 
Using existing QApplication..
Registered font families: Noto Sans, Noto Sans, Noto Sans, Noto Sans, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono, Noto Sans Mono
Failed to execute event callback < EventCallback - on_open > D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py
Traceback (most recent call last):
  File "D:\kayla\OpenPype\openpype\lib\events.py", line 163, in process_event
    callback()
  File "D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py", line 352, in on_open
    HoudiniProjectSettings().general["update_job_var_context"])
KeyError: 'update_job_var_context'
Traceback (most recent call last):
  File "D:\kayla\OpenPype\openpype\tools\workfiles\files_widget.py", line 610, in _on_workarea_open_pressed
    self.open_file(path)
  File "D:\kayla\OpenPype\openpype\tools\workfiles\files_widget.py", line 517, in open_file
    host.open_workfile(filepath)
  File "D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py", line 123, in open_workfile
    hou.hipFile.load(filepath,
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/houdini/python3.9libs\houpythonportion\ui.py", line 1119, in decorator
    return func(*args, **kwargs)
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/houdini/python3.9libs\hou.py", line 52104, in load
    return _hou.hipFile_load(self, file_name, suppress_save_prompt, ignore_load_warnings)
hou.LoadWarning: Warnings were generated during load.
Error loading: D:/test_project/new_project/ep/work/asset/new_project_ep_asset_v232.hiplc
Warning:     Problem while synchronizing child node:
             Warning:     Skipping unrecognized parameter "sepparm11".
             Skipping unrecognized parameter "tonemap".
             Skipping unrecognized parameter "tonemap_aovs".
             Skipping unrecognized parameter "tonemap_toe".
             Skipping unrecognized parameter "tonemap_shoulder".
             Skipping unrecognized parameter "tonemap_utoe".
             Skipping unrecognized parameter "tonemap_ushoulder".
             Skipping unrecognized parameter "tonemap_slope".
             Skipping unrecognized parameter "tonemap_linear".
             Skipping unrecognized parameter "tonemap_linearAngle".
             Skipping unrecognized parameter "tonemap_toeLength".
             Skipping unrecognized parameter "tonemap_shoulderLength".
             Skipping unrecognized parameter "tonemap_shoulderAngle".
             Skipping unrecognized parameter "sepparm13".
	[/out/karma_ropMain/lopnet/karma]

Setting scene FPS to 25
Using existing QApplication..
Failed to execute event callback < EventCallback - on_open > D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py
Traceback (most recent call last):
  File "D:\kayla\OpenPype\openpype\lib\events.py", line 163, in process_event
    callback()
  File "D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py", line 352, in on_open
    HoudiniProjectSettings().general["update_job_var_context"])
KeyError: 'update_job_var_context'
Traceback (most recent call last):
  File "D:\kayla\OpenPype\openpype\tools\workfiles\files_widget.py", line 610, in _on_workarea_open_pressed
    self.open_file(path)
  File "D:\kayla\OpenPype\openpype\tools\workfiles\files_widget.py", line 517, in open_file
    host.open_workfile(filepath)
  File "D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py", line 123, in open_workfile
    hou.hipFile.load(filepath,
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/houdini/python3.9libs\houpythonportion\ui.py", line 1119, in decorator
    return func(*args, **kwargs)
  File "C:\PROGRA~1/SIDEEF~1/HOUDIN~1.435/houdini/python3.9libs\hou.py", line 52104, in load
    return _hou.hipFile_load(self, file_name, suppress_save_prompt, ignore_load_warnings)
hou.LoadWarning: Warnings were generated during load.
Error loading: D:/test_project/new_project/ep/work/asset/new_project_ep_asset_v232.hiplc
Warning:     Problem while synchronizing child node:
             Warning:     Skipping unrecognized parameter "sepparm11".
             Skipping unrecognized parameter "tonemap".
             Skipping unrecognized parameter "tonemap_aovs".
             Skipping unrecognized parameter "tonemap_toe".
             Skipping unrecognized parameter "tonemap_shoulder".
             Skipping unrecognized parameter "tonemap_utoe".
             Skipping unrecognized parameter "tonemap_ushoulder".
             Skipping unrecognized parameter "tonemap_slope".
             Skipping unrecognized parameter "tonemap_linear".
             Skipping unrecognized parameter "tonemap_linearAngle".
             Skipping unrecognized parameter "tonemap_toeLength".
             Skipping unrecognized parameter "tonemap_shoulderLength".
             Skipping unrecognized parameter "tonemap_shoulderAngle".
             Skipping unrecognized parameter "sepparm13".
	[/out/karma_ropMain/lopnet/karma]

Using existing QApplication..
Failed to execute event callback < EventCallback - on_open > D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py
Traceback (most recent call last):
  File "D:\kayla\OpenPype\openpype\lib\events.py", line 163, in process_event
    callback()
  File "D:\kayla\OpenPype\openpype\hosts\houdini\api\pipeline.py", line 352, in on_open
    HoudiniProjectSettings().general["update_job_var_context"])
KeyError: 'update_job_var_context'
```

Reply:
I reverted this change as BigRoy suggested here https://github.com/ynput/OpenPype/pull/5651#discussion_r1336874211